pyproject.toml

```diff
@@ -152,6 +149,9 @@ enable_error_code = [
     "ignore-without-code",
     "unused-awaitable",
 ]
+disable_error_code = [
+    "import-untyped",
```

Comment:
It's to avoid having to install extra type modules for now (not all of them have typing and I think we can live with it for now)

Reply:
the alternative would be to suppress this warning on a per-module basis

Reply:
Mmh, maybe it's better that way actually. Technically, this error code should not overlap with something where an import fail but let's make it explicit

Reply:
Ok, installing `types-Pygments` also installs `types-docutils` which in turns makes everything a mess.

Reply:
Great, now I have more errors even without those packages. I don't know what mypy is doing or what stubs were installed :(

Reply:
Ok, so in a virtual environment it seems it's ok